### PR TITLE
ADEN-4716 - Disable ads for steam browser

### PR DESCRIPTION
**Pull request by bckatarzyna**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/config/desktop.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ define('ext.wikia.adEngine.config.desktop', [
 		log(slotName, 5, logGroup);
 		log(['getProvider', 'tracking opted ' + (isTrackingOptedIn ? 'in' : 'out')], log.levels.info, logGroup);
 
-		// If wgShowAds set to false, hide slots
-		if (!context.opts.showAds) {
+		// If wgShowAds set to false or the browser is a steam one, hide slots
+		if (!context.opts.showAds || context.opts.isSteamBrowser) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/config/desktop.js:46]: What about bidders. Should we disable them too?

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/config/desktop.js:46]: It won't handle Featured Video too

Comment (bckatarzyna) [reply extensions/wikia/AdEngine/js/config/desktop.js:46]: I will create an improvement task for bidders, featured video handled in the second commit

Comment (bckatarzyna) [reply extensions/wikia/AdEngine/js/config/desktop.js:46]: Finally there will be no task for bidders as the work required to improve it is not adequate to the importance of this feature.

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js [hunk 2]
```diff
@@ -92,6 +92,9 @@ define('wikia.articleVideo.featuredVideo.adsConfiguration', [
 			videoId = playlistItem[player.getPlaylistIndex()].mediaid,
 			videoDepth = 0;
 
+		// Disable ads for steam browser
+		if (adContext.get('opts.isSteamBrowser')) return;
```
Comment (fraszczakszymon) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js]: Avoid one-line if statements please

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**